MANIFEST.in

```diff
@@ -2,7 +2,6 @@ include README.md
 include LICENSE
 
 # Included in PyPI distribution
-include bentoml/*.py
 include bentoml/_version.py
```

Comment:
all these lines can be removed

Reply:
only `graft bentoml` is needed

Reply:
actually that's not needed neither, just suggested a change fromt he UI

Reply:
noted